src/main/java/gyro/google/cloudsql/DbReplicationCluster.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2024, Brightspot.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.cloudsql;
+
+import com.google.api.services.sqladmin.model.ReplicationCluster;
+import gyro.core.resource.Diffable;
+import gyro.core.resource.Output;
+import gyro.core.resource.Updatable;
+import gyro.google.Copyable;
+
+public class DbReplicationCluster extends Diffable implements Copyable<ReplicationCluster> {
+
+    private String failoverDrReplicaName;
+
+    // Read-only
+    private Boolean drReplica;
+    private String psaWriteEndpoint;
+
+    /**
+     * The disaster recovery (DR) replica for a primary instance.
+     * If the instance is a read replica, then the field is not set. Set this field to a replica name to designate a DR
+     * replica for a primary instance. Remove the replica name to remove the DR replica designation.
+     * Only for Enterprise Plus edition instances.
+     */
+    @Updatable
+    public String getFailoverDrReplicaName() {
```

Comment:
This will probably  be required, given it is the only settable field in this class